README.md

```diff
@@ -1 +1,76 @@
-# react-capstone-project
\ No newline at end of file
+# react-capstone-project
+In this project you can check for some nintendo characters, has a detail page for each character, this is a Single Page App Built with React, JavaScript and HTML,  it has been tested so you can also look for the testing files.
+
+## Built With:
+
+- Major languages: Javascript, HTML, CSS
+- Frameworks: React, Redux, Babel, CRA
+- Technologies used: VSC, Github
+
+## APIs used for this project : 
+
+- https://www.amiiboapi.com/
+
+## Live Demo (if available)
+
+- See live demo: https://maury-react-capstone.herokuapp.com/
+
```

Comment:
- [ ] [OPTIONAL] As suggested by the previous reviewer, it will make your README more professional if you included the presentation video.

Reply:
@AnselemOdims the link is in the readme file.
![image](https://user-images.githubusercontent.com/73630406/187759700-94641bd6-533b-4eef-b368-88348c307d74.png)

Reply:
@AnselemOdims Sorry I read it wrong, thought you guys said about the link to video, I apologize, sure, I'll add it